spec/unit/network/http/handler_spec.rb

```diff
@@ -70,7 +70,7 @@ def respond(text)
 
       expect(res[:content_type_header]).to eq("application/json")
       expect(res_body["issue_kind"]).to eq("HANDLER_NOT_FOUND")
-      expect(res_body["message"]).to eq("Not Found: No route for GET /vtest/foo")
+      expect(res_body["message"]).to eq("Not Found: No route for GET /vtest/foo #{Puppet::Network::HTTP::NOT_FOUND_ERROR_MESSAGE}")
```

Comment:
Same comment as https://github.com/puppetlabs/puppet-server/pull/371/files#r23864790

It'd be better to express an assertion against the semantics of the message, not the container of the message.